src/custom/pages/Faq/index.tsx

```diff
@@ -155,9 +155,9 @@ export default function Faq() {
           </h3>
 
           <p>
-            <a href="https://research.paradigm.xyz/MEV">Defined</a> by the Paradigm research team, Maximum Extractable
-            Value (MEV) is “a measure of the profit a miner (or validator, sequencer, etc.) can make through their
-            ability to arbitrarily include, exclude, or re-order transactions within the blocks they produce.”
+            <a href="https://arxiv.org/abs/1904.05234">Defined</a> by Phil Daian, MEV is a measure of the profit a miner
```

Comment:
It's a bit weird the link is in the "defined" word.

Maybe, we can just name the paper, or part of it and use it as the liunk `Flash Boys 2.0: Frontrunning, Transaction Reordering`

Reply:
To be honest, i left like that because that's how the previous one for paradigm was coded, so didn't want to screw the HTML i used, but yeah we can modify it like that